### ISSUE TITLE
XML Settings: Add optional master file

### ISSUE THREAD

**Issue by SendiMyrkr** [mesour-intellij-plugins/latte-pro-issues#74]
Load automatically all intellij-latte-pro.xml files is not ideal solution for many use-cases. Much better would have some master file, eg. intellij-latte-pro-specific.xml, which contains definitions to identifying files to load or even only specific tags/filter from that files. If master file i missing than behavior would be same as now.

**Comment by mabar** [comment]
Detection based on different name is problematic, vendor packages could have that file too. I would suggest to instead detect if file is defined in project or in vendor.

Inclusion/exclusion could be based on keys allowed/forbidden with list of xml configurations as a value. Either allowed or forbidden can be defined, having both is an error. If none is defined, current behavior is used.
Listed packages could be referenced via `vendor` key from config `<latte vendor="myVendor/myPackage" version="1">` tag

**Comment by SendiMyrkr** [comment]
Agree, that seems better. But what about situation where you have multiple xml configurations in project with Allow/Forbidden?

**Comment by mabar** [comment]
Check values across all files maybe? Developer should be able to keep them in sync in scope of the project.

That leads me to idea that not only vendor, but also git submodule allow/forbidden should be ignored

**Comment by SendiMyrkr** [comment]
Submodules are actually reason why i ask. Maybe add some plugin configuration with editable patterns to ignore? Or just allow choose specific file which overwrite default behaviour? When you have one file is vendor enough to ignore, when you have more it's complicated to say what is relevant and what is not. 

Choosing the master file would be best, i think. If none is chosen than plugin try to find highest with allow/forbiden and if found none then load all files.

**Comment by SendiMyrkr** [comment]
Maybe something like this inf master file

```
<include>
  <config file='../some/vendor-latte.xml' />
  <config file='../some/vendor-latte-2.xml' />
</include>
```

If you don't choose master file, than all intellij-latte-pro files are loaded otherwise only master file and files defined inside are loaded.

**Comment by mesour** [comment]
In 1.4.0 will be support for files like `intellij-latte-pro.xml`, `intellij-latte-pro-1.xml` or `intellij-latte-pro<whatever>.xml`. Prefix `intellij-latte-pro` and suffix `.xml` is important.
